Documentation/ceph-object-multisite.md

```diff
@@ -31,17 +27,19 @@ This guide assumes a Rook cluster as explained in the [Quickstart](ceph-quicksta
 
 If an admin wants to set up multisite on a Rook Ceph cluster, the admin should create:
 
-    - A [realm](/Documentation/ceph-object-multisite-crd.md)
-    - A [zonegroup](/Documentation/ceph-object-multisite-crd.md)
-    - A [zone](/Documentation/ceph-object-multisite-crd.md)
-    - An [object-store](/Documentation/ceph-object-store.md) with the zone section
+1. A [realm](/Documentation/ceph-object-multisite-crd.md)
```

Comment:
Is it normal than the first 3 are pointing to the same doc? Shouldn't the doc have a section for each?